lib/client.js

```diff
@@ -1,29 +1,22 @@
 (function (global, factory, undefined) {
 
-  if ( typeof module === 'object' && typeof module.exports === 'object' ) {
-    var ActivityStreams = require('activity-streams');
+  if ( typeof module === "object" && typeof module.exports === "object" ) {
```

Comment:
I'm not "namedropping" he made a good point that I reluctantly had to agree with at the time, however I wasn't able to find a link. It has to do with simplifying the language to it's essential parts and not using parts of the language which serve no purpose or can cause issues. Something like "What does a simplified, non-redundant, JavaScript language look like". And choosing double quotes over single-quotes was one of them. I'm working on getting linting set up across the projects but I haven't been able to do so yet.